build.gradle.kts

```diff
@@ -5,7 +5,7 @@ import org.jlleitschuh.gradle.ktlint.tasks.KtLintFormatTask
 import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
 
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "6.0.1"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "6.0.2"
```

Comment:
Bumped a handful of versions whilst I was here, but the significant changes here are pulling in the hmpps sqs starter, and the localstack testcontainers dependencies.